option/internaloption/internaloption.go

```diff
@@ -34,7 +34,19 @@ func (o defaultMTLSEndpointOption) Apply(settings *internal.DialSettings) {
 // WithDefaultMTLSEndpoint is an option that indicates the default mTLS endpoint.
 //
 // It should only be used internally by generated clients.
-//
 func WithDefaultMTLSEndpoint(url string) option.ClientOption {
 	return defaultMTLSEndpointOption(url)
 }
+
+// SkipValidation bypasses validation on ClientOptions.
```

Comment:
I'm forgetting, why don't we have an actual `internal` package for this?

Reply:
So that cloud.google.com/go can use these options as well. At least that is why the rest are here and not in `internal`